Apps/Src/main.c

```diff
@@ -137,6 +139,17 @@ void preliminaryCheck(void){
 	}
 }
 
+/** heartbeat
+ * Toggle heartbeat at visible frequency (RUN light)
+ */
+void heartbeat(void){
+	// increment heartcount variable once per while(1) loop
+	heartcount = (heartcount + 1)%(heartbeatDelay);
```

Comment:
When I compile this, I get "error: ‘heartcount’ undeclared (first use in this function);" This is because heartcount is local to main(). This can be fixed by declaring heartbeat as a static variable inside of heartbeat().